.github/workflows/test.yml

```diff
@@ -113,6 +113,8 @@ jobs:
           name: coverage-integration-test-${{ matrix.python-version }}-${{ matrix.airflow-version }}
           path: .coverage
           include-hidden-files: true
+    env:
+      ASTRO_ANYSCALE_PROVIDER_SERVICE_ID: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
```

Comment:
might be also good to suffix here python-version and airflow-version in case we add multiple values to the matrix dimensions in the future, but I guess, we're safe for now. Just had an idea :)

Reply:
Thanks, @pankajkoti , I think that's a great idea. Right now we're only running the integration tests against one version of Airflow and one version of Python, but we never know.
Just added your suggestion: [b9ef741](https://github.com/astronomer/astro-provider-anyscale/pull/62/commits/b9ef741e9a31af27c2fcab3ef3ed82ac194e28ca)